Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:20-alpine
+
+WORKDIR /app
+COPY package.json .
+
+RUN yarn install
```

```suggestion
RUN --mount=type=cache,target=/usr/local/share/.cache/yarn/v6 yarn install
```

`target=/root/.npm` for npm

https://docs.docker.com/reference/dockerfile/#example-cache-go-packages